### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by tevelee**
Hi Doug!

I've been really enjoying your series of blog posts. As someone who has been writing Swift full-time since its release, I learned a lot from them too. Your writing style is engaging, fun, and highly informative.

Unfortunately, I couldn't locate the source of the raw posts (maybe markdown files in a private repo/branch?) to implement my fixes directly. Hence, I applied the corrections on the generated HTML files, hoping they might be integrated into the main content eventually.

### CODE REVIEW

File: posts/swift-for-cxx-practitioners-type-erasure/index.html [hunk 1]
```diff
@@ -185,7 +185,7 @@
...
...things: the first is about hiding implementation details behind an abstraction barrier (e.g., we know that we have a <code>Collection</code> of <code>String</code>s, but not the specific type of the collection itself) and the other is about allowing the underlying representation to change dynamically (e.g., we can choose <code>Set&lt;String&gt;</code> or <code>[String]</code> depending on what's best for the problem at hand).</p><p>Swift has a notion of <em>opaque</em> types that let you hide the implementation type behind an abstraction barrier without allowing it to change dynamically at runtime. This gives you, as the implementer, the freedom to limit your API surface area (by not exposing some types) and evolve your implementations over time, without breaking clients. For example, let's consider implementing a generic function <code>uniqued</code> on a <code>Collection</code> that produces a new collection with duplicates removed. We could have such a function return a <code>Set</code>:</p><pre><code><span class="keyword">extension</span> <span class="type">Collection</span> <span class="keyword">where</span> <span class="type">Value</span>: <span class="type">Hashable</span> {
```
Comment (tevelee) [inline posts/swift-for-cxx-practitioners-type-erasure/index.html:188]: I corrected the word 'some' to 'specific' so it cannot be confused with the opaque type keyword `some`.
```patch
-(by not exposing some types)
+(by not exposing specific types)
```

File: posts/swift-for-cxx-practitioners-reference-types/index.html [hunk 2]
```diff
@@ -69,7 +69,7 @@
...
...e base class constructor doesn't have a fully-formed object of the subclass type, because the members of the derived class haven't been initialized yet, and it would be undefined behavior to access them. So it's not safe for the <code>Person</code> constructor to treat the object as the <code>Employee</code> instance it will eventually be. Instead, it treats the object as an instance of <code>Person</code>, calling <code>Person::greet</code> instead of <code>Employee::greet</code>, causing the difference in behavior above. This is also the reason that it's possible to get errors at runtime due to calls to abstract virtual functions in C++, even though the language prevents you from creating an instance of an abstract type. Supporting this weird semantics is also awful for C++ compiler writes; <a href="https://itanium-cxx-abi.github.io/cxx-abi/abi.html#vtable-ctor">read this</a> if you'd like to feel bad for them.</p><p>Okay, back to Swift. If we go put a call to <code>greet</code> in the initializer for <code>Person</code>, we'll get consistent output:</p><pre><code><span class="type">Hello Doug</span>, your badge number <span class="keyword">is</span> <span class="number">1</span>
```
Comment (tevelee) [inline posts/swift-for-cxx-practitioners-reference-types/index.html:72]: The diff does not show properly, but the change is one character
```patch
-C++ compiler writes
+C++ compiler writers
```

File: posts/swift-for-cxx-practitioners-extensions/index.html [hunk 3]
```diff
@@ -49,7 +49,7 @@
   <span class="keyword">var</span> isPrime: <span class="type">Bool</span> {
     <span class="keyword">switch self</span> {
       <span class="keyword">case</span> <span class="number">0</span>: <span class="keyword">return false
-      case</span> <span class="number">1</span>: <span class="keyword">return true
```
Comment (tevelee) [inline posts/swift-for-cxx-practitioners-extensions/index.html]: I do not consider 1 to be a prime number. Maybe this branch was intended for the number 2.

```swift
var isPrime: Bool {
  switch self {
    case 0, 1: return false
    case 2: return true
    default: break
  }
  ...
 ```
 
 I find it better to have an early return like
 ```swift
 guard self > 1 else { return false }
 ```
 to account for negative numbers too.

File: posts/swift-for-cxx-practitioners-type-erasure/index.html
Comment (tevelee) [inline posts/swift-for-cxx-practitioners-type-erasure/index.html:1]: In this article, I noticed this weird rendering of subsequent expressions too:
image

Comment (DougGregor) [reply posts/swift-for-cxx-practitioners-type-erasure/index.html:1]: I think I had a tab in there.

File: posts/swift-for-cxx-practitioners-type-erasure/index.html (hunk 1 above)
Comment (DougGregor) [reply posts/swift-for-cxx-practitioners-type-erasure/index.html:188]: Good idea!

**Comment by DougGregor** [comment]
Thank you! I keep all of the original sources in a private repository, and publish to this one. I've made changes for all of your suggestions, so I'll close this PR now. Thank you!

**Comment by tevelee** [comment]
Awesome! Thanks @DougGregor

**Pull request closed without merging**